.github/workflows/cargo.yml

```diff
@@ -43,3 +43,4 @@ jobs:
         # Ignore manual changes in "errno" crate until fixed in cargo-raze.
         # See: https://github.com/google/cargo-raze/issues/451
         git diff --exit-code -- ':!remote/BUILD.errno-0.2.8.bazel'
+        git diff --exit-code -- ':!remote/BUILD.cpufeatures-0.2.1.bazel'
```

Comment:
This won't work, since those are two separate tests:
1) Verify that there no differences between generated and committed code, ignoring `remote/BUILD.errno-0.2.8.bazel` - this fails, because there are manual changes in `remote/BUILD.cpufeatures-0.2.1.bazel`,
2) Verify that there no differences between generated and committed code, ignoring `remote/BUILD.cpufeatures-0.2.1.bazel` - this fails, because there are manual changes in `remote/BUILD.errno-0.2.8.bazel`.

What we need instead is a single test that verifies that there no differences between generated and committed code, ignoring both `remote/BUILD.errno-0.2.8.bazel` _and_ `remote/BUILD.cpufeatures-0.2.1.bazel`.

Reply:
It should be:
```
git diff --exit-code -- ':!remote/BUILD.cpufeatures-0.2.1.bazel' ':!remote/BUILD.errno-0.2.8.bazel' 
```

Reply:
> It should be:
> 
> ```
> git diff --exit-code -- ':!remote/BUILD.cpufeatures-0.2.1.bazel' ':!remote/BUILD.errno-0.2.8.bazel' 
> ```

Done